### PR TITLE
Add interactive plot for anomalies

### PR DESCRIPTION
**Pull request by alex-hse-repository**
**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

## Before submitting (must do checklist)

- [x] Did you read the [contribution guide](https://github.com/tinkoff-ai/etna-ts/blob/master/CONTRIBUTING.md)?
- [x] Did you update the docs? We use Numpy format for all the methods and classes.
- [ ] Did you write any new necessary tests?
- [x] Did you update the [CHANGELOG](https://github.com/tinkoff-ai/etna-ts/blob/master/CHANGELOG.md)?

## Type of Change

- [ ] Examples / docs / tutorials / contributors update
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Improvement (non-breaking change which improves an existing feature)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -74,6 +74,7 @@ flake8-docstrings = {version = "^1.6.0", optional = true}
 click = {version = "^8.0.1", optional = true}
 typer = {version = "^0.4.0", optional = true}
 semver = {version = "^2.13.0", optional = true}
+ipywidgets = "^7.6.5"
```
Comment (martins0n) [inline pyproject.toml:77]: Let's mark this as optional and add to `all` list in `[tool.poetry.extras]`

**Review by martins0n — CHANGES_REQUESTED** [review]
Could you add example to docstrings? It's not obvious now how to call this function

**Review by martins0n — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -25,6 +25,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - WandbLogger ([#71](https://github.com/tinkoff-ai/etna-ts/pull/71))
 - Correlation heatmap plotter ([#77](https://github.com/tinkoff-ai/etna-ts/pull/77))
 - Pipeline ([#78](https://github.com/tinkoff-ai/etna-ts/pull/78))
+- Interactive plot for anomalies (#[95](https://github.com/tinkoff-ai/etna-ts/pull/95))
```
[the remark this answers is not in the document]
Comment (alex-hse-repository) [reply CHANGELOG.md:28]: Sorry, overlooked)

**Review by martins0n — APPROVED** [review]
👍

**Pull request merged**